ql/src/test/queries/clientpositive/udf_substr.q

```diff
@@ -76,3 +76,8 @@ SELECT
   substr("abc 玩玩玩 abc", 5),
   substr("abc 玩玩玩 abc", 5, 3)
 FROM src tablesample (1 rows);
+
+SELECT
+  substr('ABC', cast(1 as bigint), cast(0 as bigint)),
+  substr('ABC', cast(4 as bigint))
+FROM src tablesample (1 rows);
```

Comment:
Could you please add a test when one of the long params has a value which doesn't falls into the int range? How should we handle that?